cmd/protoc-gen-grpchan/gen_test.proto

```diff
@@ -2,6 +2,8 @@ syntax = "proto3";
 
 package main;
 
+option go_package = ".;main";
```

Comment:
This is gross, but this is what the latest `protoc-gen-go` wants: it wants every proto to have a `go_package`. The `.` means assume "current directory" as the go package path. The part after the semicolon is the actual package name that appears in `package` decls in the generated files.

Reply:
Does that mean moving to this version means transitively upgrading and fixing all our own proto files?

Reply:
No. This is because I opted this repo into the new version of `protoc-gen-go`.

Before _we_ update to the new `protoc-gen-go`, I think we will in fact have to do this. (Which is why we have the weird hybrid build -- where we use the newer runtime version but older version of protoc plugin.)